Tests/SwiftSyntaxTest/AbsolutePosition.swift

```diff
@@ -13,22 +13,22 @@ public class AbsolutePositionTestCase: XCTestCase {
     XCTAssertNoThrow(try {
       let source = try String(contentsOf: getInput("visitor.swift"))
       let parsed = try SourceFileSyntax.parse(getInput("visitor.swift"))
-      XCTAssertEqual(parsed.position.byteOffset, 0)
-      XCTAssertEqual(parsed.eofToken.positionAfterSkippingLeadingTrivia.byteOffset,
-                  source.count)
-      XCTAssertEqual(parsed.position.byteOffset, 0)
-      XCTAssertEqual(parsed.byteSize, source.count)
+      XCTAssertEqual(0, parsed.position.utf8Offset)
+      XCTAssertEqual(source.count,
+        parsed.eofToken.positionAfterSkippingLeadingTrivia.utf8Offset)
+      XCTAssertEqual(0, parsed.position.utf8Offset)
```

Comment:
Can we write the above two `XCTAssertEqual` to `0` the other way round, i.e. `0` as the second parameter?

Reply:
Done.